<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WStatisticUI</class>
 <widget class="QWidget" name="WStatisticUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>189</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistic</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="coverage_label_p">
     <property name="text">
      <string>Coverage:</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QProgressBar" name="coverage_bar_p">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="coverage_value_p">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="coverage_level1_label_p">
     <property name="text">
      <string>Coverage:</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QProgressBar" name="coverage_level1_bar_p">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="coverage_level1_value_p">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
